packages/compass-data-modeling/src/components/diagram-editor.tsx

```diff
@@ -77,6 +86,27 @@ const ErrorBannerWithRetry: React.FunctionComponent<{
   );
 };
 
+function getFieldTypeDisplay(field: MongoDBJSONSchema) {
+  if (field.bsonType === undefined) {
+    return 'unknown';
+  }
+
+  if (typeof field.bsonType === 'string') {
+    return field.bsonType;
+  }
+
+  const typesString = field.bsonType.join(', ');
+
+  // We show `mixed` with a tooltip when multiple bsonTypes were found.
+  return (
+    <Tooltip justify="end" spacing={5} trigger={<div>mixed</div>}>
```

Comment:
```suggestion
    <Tooltip justify="end" spacing={5} trigger={<div>(mixed)</div>}>
```
To match designs: https://www.figma.com/design/w51HGIb2PDdyZqyCOMMypr/-INIT-592--Data-Model-Design-Experience?node-id=69-18771&m=dev